components/authentication/authUtils.js

```diff
@@ -67,7 +66,7 @@ const passwordAPIChecker = async password => {
   let range = hashed.slice(0, 5);
   let suffix = hashed.slice(5);
 
-  let res = await axios(`https://api.pwnedpasswords.com/range/${range}`);
+  let res = await axios.get(`https://api.pwnedpasswords.com/range/${range}`, { withCredentials: false });
```

Comment:
Does this still work if you remove the below `{ withCredentials: true }` lines and leave the previous line of `axios.defaults.withCredentials = true`

Reply:
as long as pwnedpasswords api uses `{withcredentials:false}`

Reply:
just didnt want it globally just incase

Reply:
Yeah that's fair. If you used my example to do this I didn't do it in the best way. I should've created an axios instance, which should've been done in this project. I'm not too nitpicky with it since we are in the final stages so this looks good.